### ISSUE TITLE
Consider integration with ioos checker plugin

### ISSUE THREAD

**Issue by pp-mo** [SciTools/ugrid-checks#44]
See https://github.com/ioos/cc-plugin-ugrid

@ocefpaf suggests a code sprint : https://github.com/ioos/ioos-code-sprint/issues/6
Potentially, this would be [April 25 - 28, 2022](https://github.com/ioos/ioos-code-sprint)

**Comment by ocefpaf** [comment]
Note that it would be nice to have you all at the code sprint for other topics unrelated to ugrid.  Hopefully the remove alternative will make it easier to attend. However, it would be awesome to have you for the in person event.

**Comment by pp-mo** [comment]
@ocefpaf thanks
TBH I think Chicago is probably a bit of a stretch for us right now.
But some involvement should certainly be possible.  

Noticing in
 >  "General Python :: strategies for querying/visualizing gridded data (pygeoapi, stac, xarray) as outputs get larger (higher resolution, more variables)" 

Maybe we could present on Geovista / PyVista ?  
That's looking like our solution of choice here (though our Scientists are really only just getting started on using 3d viz )

( 
N.B. : 
Geovista is still a bit obscure + the strapline doesn't clearly sell it IMHO
  - basically, **2d earth-surface helpers add cartographic analysis and cartographic to PyVista**
proper docs in the pipeline, but bjlittle has just promosed to slightly improve it  -- ?? didn't you @bjlittle ?!?
)

**Comment by pp-mo** [comment]
P.S. 
> Geovista is still a bit obscure + the strapline doesn't clearly sell it IMHO

@trexfeathers suggests this from the Iris docs : https://scitools-iris.readthedocs.io/en/latest/further_topics/ugrid/partner_packages.html#geovista
That is perhaps more understandable.

**Comment by ocefpaf** [comment]
The "main" event is focused on Software discussion/hacking but there will be lightning talks and I believe geovista is definitely worth mentioning in one of them.

**Comment by pp-mo** [comment]
If there's still space, then we could inject a lightning talk on GeoVista, as you suggest.
The best person is probably @bjlittle - currently on holiday but back next week, I believe.
Or @trexfeathers has also written [something maybe suitable](https://github.com/SciTools-incubator/presentations/tree/main/ngms_champions_2022-04-12/geovista_notebook), which we delivered a couple of weeks back.

**Comment by ocefpaf** [comment]
I believe there is. Did you all register for it? Are you in touch with @MathewBiddle?

**Comment by pp-mo** [comment]
> I believe there is. Did you all register for it? Are you in touch with @MathewBiddle?

Actually no, I wasn't sure who to approach.
I don't have access to the Slack account, if that is where things are happening, since it is available only to specified domains and I couldn't find a visible contact to apply for access.
 -- and I left it much too late, of course :wink:

### Regarding possible use of the ugrid-checks code
I have already created a proof-of-concept solution (incomplete, and not yet visible here) to provide checking results in a suitable form for inclusion in an IOOS-style Report object.  
So, if anyone is interested, I think we could collaborate on creating an ioos plugin to integrate ugrid-checks via such an interface.
It makes sense to me, that we would want the plugin in its own project/repo.  But of course, the sprint may simply not be a very good time, since the time difference can be a bit of an obstacle. 
Also, I still remain to be convinced that this is particularly **_wanted_** -- but the interfacing problem did pique my interest  !

**Comment by ocefpaf** [comment]
Folks are discussing that today and tomorrow during the compliance-checker sprint. If you reach out to Matt he will send you the URLs you need to connect. It is be zoom for discussion and slack for async/pasting text.
